modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -29,13 +32,28 @@ $ oc create serviceaccount thanos && oc describe serviceaccount thanos
 Name:                thanos
 Namespace:           openshift-ingress-operator
 Labels:              <none>
-Annotations:         <none>
-Image pull secrets:  thanos-dockercfg-b4l9s
-Mountable secrets:   thanos-dockercfg-b4l9s
-Tokens:              thanos-token-c422q
+Annotations:         openshift.io/internal-registry-pull-secret-ref: thanos-dockercfg-kfvf2
+Image pull secrets:  thanos-dockercfg-kfvf2
+Mountable secrets:   thanos-dockercfg-kfvf2
+Tokens:              <none>
 Events:              <none>
 ----
 
+. Enter the following command to view the current cluster version and status:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME      VERSION                              AVAILABLE   PROGRESSING   SINCE   STATUS
+version   {product-version}.0-0.nightly-{build-metadata}   True        False         40m     Cluster version is {product-version}.0-0.nightly-{build-metadata}
+----
+
```

Comment:
Hmm where did this come from? The {product-version} (and others) variable substitution doesn't work here in the preview.

I don't see the need for listing the cluster version in this procedure, what was the reason for adding?

Reply:
Hi @gcs278 . Thanks for the comment. I got this output from Shudi's Jira: https://issues.redhat.com/browse/OCPBUGS-39242.

Any commands from this Jira worth implementing?

Reply:
I don't think Shudi intended for this to be in the docs, I think he was just showing the cluster as a part of the bug/reproducer. I don't think it adds value here.

Nope, I don't think we need anything more than what you have.

Reply:
Thanks, Grant. I removed this section.